### DISCUSSION TITLE
How do I configure the Web Search

### DISCUSSION THREAD

**Discussion by CyberDoyle** [pop-os/launcher#3] [Q&A]
I want to change the activation of a google search from the launcher from 'google {my query}' to just 'g {my query}'. 

I have looked at "/usr/bin/env" file as following the github tree
to "plugins/src/plugins/web/web.js", the first line is "#!/usr/bin/env gjs"
But the "env" file is uneditable. I can't find the path to a config file online for this launcher.

-----

Coming from Wox launcher on windows, its gui is fairly intuitive to make this change. It has a Web Search plugin similar to Pop Os' but you can easily add, change and remove websites at will.
I would like a launcher ideally that works as good as or similar to Alfred on Mac. Wox on windows was decent. It did most things well but lacked some features and there was a few bugs I wasn't a fan of.
I have heard good things about Rofi on Linux and as far as I am aware its not too hard to customize. But I like this Pop!_Os launcher and I think it is very promising if the plugins and customizability becomes easier/better documented.

-----

I'm new to linux, and I don't have much coding experience yet. Just made the transition on my main workstation from Windows 10 to Pop.

**Comment by mmstick** [comment]
`/usr/bin/env gjs` is an explicit way of telling the interpreter to invoke the gjs executable located in a `PATH` defined in the environment.

The web plugin is not configurable right now.

**Reply by CyberDoyle** [reply]
Ok, thank you for the response.
Just wondering if you have any idea when that functionality might become available.

**Reply by mmstick** [reply]
This repository was created on Friday. Once it has been released, and I take care of some GNOME 40 issues in Pop Shell, I can look into adding support for plugin configurations, and a plugin configuration schema for generating settings GUIs.

**Reply by CyberDoyle** [reply]
Ok, thanks for the response. I look forward to the update.

**Reply by mmstick** [reply]
The web plugin is configurable when it lands. Default config looks like so: https://github.com/pop-os/launcher/blob/master/plugins/src/web/config.ron
